sections/Deals/CreateDeal.tsx

```diff
@@ -277,7 +277,7 @@ const CreateDeal: FC<CreateDealProps> = ({ poolAddress, purchaseToken }) => {
 				const poolSupply = wei(supply, decimals);
 				setTotalPoolSupply(poolSupply);
 				if (allocation === Allocation.MAX) {
-					formik.setFieldValue('purchaseTokenTotal', poolSupply.toString());
+					formik.setFieldValue('purchaseTokenTotal', removeZeroes(poolSupply.toString()));
```

Comment:
Are we sure there're no side effects? The purchaseTokenTota valuel is using to [call the createDeal function](https://github.com/AelinXYZ/aelin-ui/blob/dev/sections/Deals/CreateDeal.tsx#L138). I would say that we need to format the number when the value is displayed.

Reply:
I dont see why removing insignificant zeros could cause a side effect. We are already converting it to string...Besides this bug was originated bc we were removing zeros in the input, after onChange. However I see what you mean, so it would be worthwhile to run a e2e test here